components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -65,6 +65,7 @@ class GeckoEngineSession(
     internal lateinit var geckoSession: GeckoSession
     internal var currentUrl: String? = null
     internal var scrollY: Int = 0
+    internal var firstContentfulPaint = false
```

Comment:
nit: Maybe we could improve the name to something like`canCaptureThumbnails` or something else that is more descriptive ?

Reply:
Hm I don't want to change the name as it's from GV and not specific to capturing thumbnails. So, I've added a description instead.

Reply:
👍